docs/table/access_table.rst

```diff
@@ -288,6 +288,19 @@ selected rows or columns.  ::
        9.000  11
       12.000  14
 
+We can select rows from a table using conditionals to create boolean masks. A
+table indexed with a boolean array will only return rows where the mask array
+is equal to `True`. Different conditionals can be combined using the bitwise
```

Comment:
Suggest writing as below.  Slightly fussy, but by analogy with PEP8 one writes `is True` not `is equal to True`.
```
element is `True`.  Different ...
```